types/webpack/index.d.ts

```diff
@@ -598,7 +601,358 @@ declare namespace webpack {
             portableRecords?: boolean;
         }
     }
+    namespace compilation {
+        class Asset {
+        }
+
+        class Module {
+        }
+
+        class Record {
+        }
+
+        class Chunk {
+            constructor(name: string);
+            id: any;
+            ids: any;
+            debugId: number;
+            name: any;
```

Comment:
string?

Reply:
I've been using `any` in a few places like this as a stop gap till I am able to figure out what all the types are...